hist/hist/test/test_Project3D_name.cxx

```diff
@@ -0,0 +1,14 @@
+// test TH3::Project3D name
+
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+using ::testing::MatchesRegex;
+
+#include "TH2D.h"
+#include "TH3D.h"
+
+TEST(TH3, Project3D) {
+   auto h3 = new TH3D("h3","Title",10,0.,1.,10,0.,1.,10,0.,1.);
+   auto h2 = (TH2D*)h3->Project3D("Projection_xy");
+   EXPECT_THAT(h2->GetName() , MatchesRegex("h3_Projection_xy"));
```

Comment:
Does "MatchesRegex" mean that only the string "h3_Projection_xy" it it's matched? Identity should be verified in this case I think.

Reply:
I'll check. That was the way I found to compare strings.

Reply:
One also can use HasSubstr it seems ...